### PR TITLE
[bug] Harmonize dialog boxes across the application

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-30: [Harmonize dialog boxes across the application](https://issues.openmrs.org/browse/PDMO-30)

## SUMMARY:
Harmonize dialog boxes across the application

### CODE REVIEW

**Comment by jomadoye** [comment]
Good job @andela-cuchendu

**Comment by kingisaac95** [comment]
For code readability, you may want to consider changing the indentations of the buttons.
Instead of having this:
```
<button
    type="button" className="btn btn-danger form-control cancelBtn"
    data-dismiss="modal"
>
   Close
</button>
```
you can indent it to look like this:
```
<button
    type="button"
    className="btn btn-danger form-control cancelBtn"
    data-dismiss="modal"
>Close</button>
```
so that the text to be displayed on the button can be seen clearly without confusing it for a prop.

Apart from that, the work looks good to me. Good job @andela-cuchendu 👍

File: app/js/components/subAddress/addressForm.jsx [hunk 1]
```diff
@@ -169,20 +150,40 @@ export default class AddressForm extends React.Component {
...
+                <div className="modal-header">
+                  <button type="button" className="close" data-dismiss="modal">&times;</button>
+                  <h4 className="modal-title">Confirm Delete</h4>
                 </div>
-              </ModalDialog>
-            </ModalContainer>
-          }
+                <div className="modal-body">
+                  <p> Are you sure want to delete this address?</p>
+                </div>
```
Comment (jacksono) [inline app/js/components/subAddress/addressForm.jsx]: Seems theres a typo on the prompt message. "Are you sure **you** want to delete this address?

File: app/js/components/obs.jsx [hunk 2]
```diff
@@ -446,29 +337,43 @@ export default class Obs extends React.Component {
...
+          <div className="modal fade" id="myModal" role="dialog">
+          <div className="modal-dialog">
+            <div className="modal-content">
+              <div className="modal-header">
+                <button type="button" className="close" data-dismiss="modal">&times;</button>
+                <h4 className="modal-title">Confirm Delete</h4>
+              </div>
+              <div className="modal-body">
+                <p> Are you sure want to Delete this Observation?</p>
               </div>
```
Comment (jacksono) [inline app/js/components/obs.jsx]: Seems theres a typo on the prompt message. "Are you sure **you** want to delete this Observation?

**Review by jacksono — CHANGES_REQUESTED** [review]

**Comment by jacksono** [comment]
Good job 👍

**Comment by chibujax** [comment]
All done. Thank you  @jacksono and @kingisaac95

**Pull request merged**